### PR TITLE
Fix: #6. Use Structured Fields.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- uses structured fields
- based on @ucarion preliminary work: thanks!!!

## It is done

- referencing the RFC
- distinguishing sf-item and sf-integer, since sf-integer do not support parameters whereas sf-item does 
- retaining the intermediate syntaxes (quota-policy, quota-units, service-limit, ...)

### CODE REVIEW

**Review by unleashed — COMMENTED** [review]
Looks good - except the formatting of `w`, which I'll wait for confirmation it looks fine in the rendered document.

**Review by unleashed — APPROVED** [review]

**Comment by unleashed** [comment]
LGTM, waiting on ack from @mnot before merging.

File: draft-ietf-httpapi-ratelimit-headers.md
Comment (mnot) [inline draft-ietf-httpapi-ratelimit-headers.md:267]: It's not necessary to say this; SF takes care of it already.

```suggestion
```

**Comment by ioggstream** [comment]
@mnot thanks for reviewing! Can we merge?

**Comment by mnot** [comment]
It's your draft :)

**Comment by ioggstream** [comment]
I take it for a YES :D

**Pull request merged**
